packages/components/src/components/drawer/drawer.ts

```diff
@@ -44,43 +43,33 @@ import type { CSSResultGroup } from 'lit';
  * @csspart overlay - The overlay that covers the screen behind the drawer.
  * @csspart panel - The drawer's panel (where the drawer and its content are rendered).
  * @csspart header - The drawer's header. This element wraps the title and header actions.
- * @csspart header-actions - Optional actions to add to the header. Works best with `<sd-icon-button>`.
+ * @csspart header-content - Optional actions to add to the header. Works best with `<sd-icon-button>`.
```

Comment:
We don't have a sd-icon-button. Furthermore, AFAIK we even don't provide the whole part or slot, as we just name it `header`, I believe?

Reply:
Should I just remove it then? We had a note left from refinement so I just renamed it but feels like there's no real use for it.